### ISSUE TITLE
the shape of label and the output of the layer is different

### ISSUE THREAD

**Issue by MisayaZ** [sunshineatnoon/Darknet.keras#3]
I run the TrainYolo.py and using Tensorflow as keras backend, howerver i get exception:
Exception: Error when checking model target: expected dense_3 to have shape (None, 1470) but got array with shape (16, 1225)
the shape of label and the output of the layer is different.

**Comment by sunshineatnoon** [comment]
Sorry, I haven't implemented the train process yet.
